src/super_gradients/common/environment/env_variables.py

```diff
@@ -45,5 +45,9 @@ def HYDRA_FULL_ERROR(self) -> Optional[str]:
     def HYDRA_FULL_ERROR(self, value: str):
         os.environ["HYDRA_FULL_ERROR"] = value
 
+    @property
+    def SUPER_GRADIENTS_LOG_DIR(self) -> str:
+        return os.getenv("SUPER_GRADIENTS_LOG_DIR", default="~/sg_logs")
```

Comment:
I suggest using `str(Path.home())` as a default value here, this makes expanduser call in other places unneecessary.